spec/acceptance/instance_spec.rb

```diff
@@ -35,6 +35,13 @@ def get_instance(name)
         :volume_size => 8,
       }
 
+
+      # The value for this ENV var must be an existing IAM role in your Amazon account
+      # the IAM role must be available in the acceptance test region.
+      #
+      # When testing IAM roles you need to provide the IAM role name and the coresponding ARN.
+      @config[:iam_instance_profile_name] = ENV['IAM_ROLE_NAME'] if ENV['IAM_ROLE_NAME'] && ENV['IAM_ROLE_ARN']
```

Comment:
@garethr I am interested in your view on if you think it is necessary to test both the iam_instance_profile_name and iam_instance_profile_arn code paths when creating an instance?

Currently I have the acceptance test setup to create the instance with the value from iam_instance_profile_name and validate the IAM role based on the value of the iam_instance_profile_arn.

This would save provisioning an additional instance just to confirm setting the IAM role via the iam_instance_profile_arn works.

Thoughts?